README.md

```diff
@@ -40,10 +40,10 @@ from the main branch.
 The table below summarizes the options regarding the images, manifests, documentation and examples and gives your links
 to the correct versions:
 
-| Version        | Description                              | Installation Manifests                                                            | Documentation and Examples                                                                                                                                             |
-|----------------|------------------------------------------|-----------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Latest release | For production use                       | [Manifests](https://github.com/nginxinc/nginx-gateway-fabric/tree/v1.0.0/deploy). | [Documentation](https://github.com/nginxinc/nginx-gateway-fabric/tree/v1.0.0/docs). [Examples](https://github.com/nginxinc/nginx-gateway-fabric/tree/v1.0.0/examples). |
-| Edge           | For experimental use and latest features | [Manifests](https://github.com/nginxinc/nginx-gateway-fabric/tree/main/deploy).   | [Documentation](https://github.com/nginxinc/nginx-gateway-fabric/tree/main/docs). [Examples](https://github.com/nginxinc/nginx-gateway-fabric/tree/main/examples).     |
+| Version        | Description                              | Installation Manifests                                                            | Documentation and Examples                                                                                                                                                     |
+|----------------|------------------------------------------|-----------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Latest release | For production use                       | [Manifests](https://github.com/nginxinc/nginx-gateway-fabric/tree/v1.1.0/deploy). | [Documentation](https://github.com/nginxinc/nginx-gateway-fabric/tree/v1.1.0/site/content). [Examples](https://github.com/nginxinc/nginx-gateway-fabric/tree/v1.1.0/examples). |
+| Edge           | For experimental use and latest features | [Manifests](https://github.com/nginxinc/nginx-gateway-fabric/tree/main/deploy).   | [Documentation](https://docs.nginx.com/nginx-gateway-fabric/). [Examples](https://github.com/nginxinc/nginx-gateway-fabric/tree/main/examples).                                |
```

Comment:
Are the documentation links accurate here? It seems like the "live" link should be to the URL whereas the edge one should be to the repository, assuming a model where the `main` branch is work in progress while a release branch is production-ready.

Reply:
Good catch, I accidentally swapped them.

Reply:
Fixed!